### DISCUSSION TITLE
ini settings library

### DISCUSSION THREAD

**Discussion by Jagge** [karttapullautin/karttapullautin#164] [Ideas]
Not a new feature really, but would it be useful to have a folder/library of ini settings pre-tuned for certain forest/vegetation types? Se evenually there would ne inin files like "South Finland basemap", "Andalucia", "New England mapant", "Trondheim", "Alaska", "Colorado/Wyoming", "California" so when making map for a new area one could try some files/settings to figure the best and if it feels like it shoudl be something between two, you could possibly see look at the difference figure how to find settings halfway in between. 

This would be mostly just users/mappers sharing their settings and maybe explaining what kind of area/vegetation/data it is for and is it for basemap or direct use. I think now we have just onini e default ini (with not that good defalt veg settings really). And not coding anything really.

**Comment by stefankinell** [comment]
I would like that. And perhaps also some discussions/insight into how the different settings work, I.e. what they influence. 

I have today 12 different settings I use for Sweden, which I have calculated somewhat. But they are really only changing the base of what becomes light green. And shifting the rest a bit upwards. I think I could do so much more to optimize these settings.

**Reply by Jagge** [reply]
I use ini with abut 30 green tones. Then count the shade I want to become as a light green, medium green and dark. Then either edit ini accordingly as a 3 greens ini or I simply correct my green tones with photo editor. But the idea with these shared ini files would be having all kinds of settings to detect bad vegetation / not bad vegetation from a various typed of point cloud and terrains. Getting the threshold for tones right is pretty trivial compared to that.

**Comment by rphlo** [comment]
This would probably help newcomers, not sure where it would belong though, an example ini directory in the release folder of just a wiki where users would share what they have used...
